uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -220,9 +220,7 @@ sub extractAndParseTarchive {
 sub determineSubjectID {
 
     my $this = shift;
-    my ($scannerID,$tarchiveInfo,$to_log) = @_;
-    my $tarchive_srcloc = $tarchiveInfo->{'SourceLocation'};
-    my $upload_id = getUploadIDUsingTarchiveSrcLoc($tarchive_srcloc);
+    my ($scannerID,$tarchiveInfo,$to_log, $upload_id) = @_;
```

Comment:
what will happen to the scripts that currently call this function while passing the $tarchive_srcloc? 
i.e. tarchiveLoader and tarchive_validation.pl scripts.

Reply:
oups... Very valid point that I oversaw. Fixing it for all the functions that were changed in MRIProcessingUtility.pm.